draft-ietf-quic-recovery.md

```diff
@@ -506,8 +506,8 @@ largest_sent_before_rto:
 : The last packet number sent prior to the first retransmission
   timeout.
 
-time_of_last_sent_packet:
-: The time the most recent packet was sent.
+time_of_last_sent_retransmittable_packet:
```

Comment:
I'm trying to remove the use of retransmittable from this document in a different PR, so I'd prefer not to add it here.  Can you change the text below to be "The time the most recent congestion-controlled packet was sent."?  For the variable rename, is there something shorter we could come up with that doesn't use the word retransmittable?

Reply:
I used retransmitted here since your PR isn't merged yet.

Maybe we can call these packets ackable instead of congestion controlled? I'm not sure if I really like this term, since ACK-only packets can also be acked (later, as soon as a retransmittable packet is received), but I can't think of anything better right now.

Reply:
time_of_last_ack_eliciting_packet?